task_scheduler/utils/logger.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+
+class CustomLogger(logging.Logger):
+    # Custom Logger should be created when having an app taht has many modules
+    # or your app uses part time modules
```

Comment:
fyi.  this was only for an example. Please use docstrings to document the goal of this class